### PR TITLE
Use pe_puppetserver_gem to install carrier-pigeon

### PR DESCRIPTION
**Pull request by joshsamuelson**
@binford2k can you review and confirm that this will actually work for the student masters?  I just moved the carrier-pigeon gem to the classroom::master::ircd class.  Do I need to do something more extensive and create a 3rd kind of classification for student masters or will this work for the lab?

### CODE REVIEW

**Comment by binford2k** [comment]
This will work, but only incidentally, because we happen to be running the IRC server on the same host as the Puppet master. I'd prefer they were in separate classes, though the specific implementation isn't as important to me.

**Comment by joshsamuelson** [comment]
Ah, good point.  Don't merge yet, I'll do it the right way and push again.  This just needs to get merged and released before the next offline practitioner, so I'd rather take the time to do it right.

File: metadata.json [hunk 1]
```diff
@@ -25,26 +25,27 @@
...
+    {"name":"jamtur01/irc",                   "version_requirement":">= 0.0.7"},
+    {"name":"puppetlabs/apache",              "version_requirement":">= 1.1.1"},
+    {"name":"puppetlabs/concat",              "version_requirement":">= 1.0.4"},
+    {"name":"puppetlabs/git",                 "version_requirement":">= 0.2.0"},
+    {"name":"puppetlabs/haproxy",             "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/inifile",             "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/mysql",               "version_requirement":">= 2.3.1"},
+    {"name":"puppetlabs/ntp",                 "version_requirement":">= 3.2.0"},
+    {"name":"puppetlabs/stdlib",              "version_requirement":">= 4.0.0"},
+    {"name":"puppetlabs/pe_gem",              "version_requirement":">= 0.0.1"},
```
Comment (casharma) [inline metadata.json]: You can drop this dependency now.

Comment (binford2k) [inline metadata.json]: if we ever install `eyaml` or the like--something with a CLI component--then we'll need it. Might be good to keep around.

**Pull request merged**